sdk/tools/asmpp/asmpp.cpp

```diff
@@ -0,0 +1,1152 @@
+/*
+ * PROJECT:     ReactOS host tools
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     ASM preprocessor
+ * COPYRIGHT:   Copyright 2021 Timo Kreuzer <timo.kreuzer@reactos.org>
+ */
+
+// Optimize even on debug builds, because otherwise it's ridiculously slow
+#ifdef _MSC_VER
+#pragma optimize("gst", on)
+#pragma auto_inline(on)
+#else
+#pragma GCC optimize("O3,inline")
+#endif
```

Comment:
Can this be added within the CMakeLists file?

Reply:
Probably, but what's the argument for doing that?